### PR TITLE
Update README.Rmd

### PR DESCRIPTION
**Pull request by katieroserice**
Added description. 
(I am an rOpenSci intern editing README.Rmds.)

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -20,7 +20,11 @@ knitr::opts_chunk$set(
 
 Wrapper for the USGS Bison API.
 
-## Info
+## Description
+
+USGS Biodiversity Information Serving Our Nation (BISON) is a unique, web-based Federal mapping resource that provides access to georeferenced (those with latitude and longitude coordinates) and non-georeferenced data describing the occurrence or presence of terrestrial and aquatic species recorded or collected by a person (or instrument) at a specific time in the United States and its Territories. Each record in a species occurrence dataset available in BISON will typically consist of a scientific name (genus and specific epithet), a date, and one or more geographic references such as a state name, county name, and/or decimal latitude and longitude coordinates. In addition to these typical data fields, species occurrence datasets often include many other data fields that describe each species occurrence event in more detail.
+
+rbison allows one to pull species occurrence data from these datasets, inspect species occurance summaries, and then map species occurance within the US, within the contiguous 48 states, and/or at county or state level.
```
Comment (sckott) [inline README.Rmd]: @katieroserice there's a typo, `occurance` misspelled (a few of them)

File: README.Rmd [hunk 2]
```diff
@@ -20,7 +20,11 @@ knitr::opts_chunk$set(
 
 Wrapper for the USGS Bison API.
 
-## Info
+## Description
+
+USGS Biodiversity Information Serving Our Nation (BISON) is a unique, web-based Federal mapping resource that provides access to georeferenced (those with latitude and longitude coordinates) and non-georeferenced data describing the occurrence or presence of terrestrial and aquatic species recorded or collected by a person (or instrument) at a specific time in the United States and its Territories. Each record in a species occurrence dataset available in BISON will typically consist of a scientific name (genus and specific epithet), a date, and one or more geographic references such as a state name, county name, and/or decimal latitude and longitude coordinates. In addition to these typical data fields, species occurrence datasets often include many other data fields that describe each species occurrence event in more detail.
```
Comment (sckott) [inline README.Rmd]: I'd remove `unique` in the first line

File: README.Rmd (hunk 1 above)
Comment (sckott) [inline README.Rmd]: @katieroserice thoughts

File: README.Rmd (hunk 2 above)
Comment (sckott) [inline README.Rmd]: @katieroserice thoughts

Comment (katieroserice) [inline README.Rmd]: yes i agree.

File: README.Rmd (hunk 1 above)
Comment (katieroserice) [inline README.Rmd]: I think "occurrence" is the correct spelling.

File: README.Rmd (hunk 2 above)
Comment (sckott) [inline README.Rmd]: cool, can you make that change then?

**Pull request closed without merging**